javatests/arcs/android/host/parcelables/ParcelablePlanTest.kt

```diff
@@ -17,8 +17,7 @@ import arcs.core.common.ArcId
 import arcs.core.data.EntityType
 import arcs.core.data.FieldType
 import arcs.core.data.Schema
-import arcs.core.data.SchemaDescription
-import arcs.core.data.SchemaFields
+simport arcs.core.data.SchemaFields
```

Comment:
Typo at start of line

Reply:
this is truly embarrassing and explains all the failures... fixed.
sorry and thanks.